resources/org/centos/contra/Infra/topologyTemplates/openshift.template

```diff
@@ -0,0 +1,23 @@
+${index}_${providerType}:
```

Comment:
this file may have more iterations to follow, depending on how well the LP provider is tested, I'll update on my working branch and squash all the final commits when we have the image updated